guide/payments/overview.md

```diff
@@ -0,0 +1,77 @@
+title: Transaction overview
+description: Principles to strive for while building bitcoin applications for end-users.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/overview/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/overview/page-overview.svg
+---
+
+# Transactions overview
+
+Bitcoin is a peer-to-peer push-payment system. This means that you are able to send, or *push*, bitcoin to any address at your will, at any time, without passing through a third party.
+
+
+This is radically different from the legacy financial system where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants etc.). When you make a payment it will pass through systems that might delay, control or block the payment. 
+## Bitcoin transactions
+
+The most common bitcoin transactions simply contain instructions to move funds from one address to another. Other types of transactions include creating multi-sig wallets or recording data on the blockchain.
+
+Once created, a transaction needs to be signed with the private key(s) of the sender(s), submitted to a node on the network for validation and propagation to other nodes. It then waits to be verified and included in a newly mined block, after which point it will be recorded on the blockchain. For this service, the sender will pay a fee for every transaction which goes to the node who mined the block as a reward.
+
+Technically, the transaction is data that contains the necessary information to execute on the blockcain. The principal properties are:
+
+- Amount - the amount of bitcoin, in satoshis
+- Inputs - the source(s)
+- Outputs - the destination(s)
+- Version - the code version this transaction follows
+- Lock time - time until the transaction should be executed, optional
+
+### Input and outputs
+
+In a Bitcoin wallet, funds are often not held in a single address, but more commonly in one address per transaction where you received bitcoin. When you are making a transaction you need to specify which of your addresses should be the source of the funds. If you need to spend more than what a single address holds, you can specify several. These are called *inputs* to the transaction.
+
+Likewise, you need to specify the destination address, or addresses for the transaction. These are called *outputs*. Should there be more bitcoin in the inputs than are needed for the transaction, a new address will be created in your wallet for the change, often called a *change output*.
+
+### Transaction fee
+
+Every transaction needs to pay a fee to incentivises miners to include it in a block. There is no fixed fee for making a transaction as it depends on the amount of data it includes, the amount of other transactions that are trying to get verified, and how much each submitter is prepared to pay at any given time. Miners typically pick the transactions that will earn them the highest reward to include in a block.
```

Comment:
Since we use American vs British English in the guide, just for consistency..
```suggestion
Every transaction needs to pay a fee to incentivizes miners to include it in a block. There is no fixed fee for making a transaction as it depends on the amount of data it includes, the amount of other transactions that are trying to get verified, and how much each submitter is prepared to pay at any given time. Miners typically pick the transactions that will earn them the highest reward to include in a block.
```